fiat-web/src/main/java/com/netflix/spinnaker/fiat/config/FiatConfig.java

```diff
@@ -88,6 +90,15 @@ DefaultApplicationResourceProvider applicationProvider(
         properties.isAllowAccessToUnknownApplications());
   }
 
+  @Bean
+  @ConditionalOnProperty(
+      value = "fiat.service-account-resource-provider.default.enabled",
```

Comment:
I tried a few things to make this just `@ConditionalOnBean` but we would have to rework a bunch of our things away from initializing via `@ComponentScan` of `com.netflix.spinnaker.config. If everything was loaded via autoconfigurations I think it could work, but I don't really want to boil that ocean at the moment.

Reply:
Yeah I've tried this before and thrown my hands up.